Settings-default.toml

```diff
@@ -76,6 +76,7 @@ dataset_defs_path = "./test_data/dataset_defs"
 provider_defs_path = "./test_data/provider_defs"
 layer_defs_path = "./test_data/layer_defs"
 layer_collection_defs_path = "./test_data/layer_collection_defs"
+ml_model_defs_path = "./test_data/ml_model_defs"
```

Comment:
Rather put it under `pro.ml.xgboost.model_path`

Reply:
https://github.com/geo-engine/geoengine/pull/639/commits/2e54047e11dfc53922b2a0b5103287109b43b214

Reply:
https://github.com/geo-engine/geoengine/pull/639/commits/03f16457a9068a96679fa8c005c00009fe606e37